acceptance/ui/features/data/default/hosts.json

```diff
@@ -25,7 +25,7 @@
       "hostID": 4982,
       "memory": 10737418240,
       "memoryGB": "10.00 GB",
-      "ramGB": "5.34 GB",
+      "ramGB": "5737418240",
```

Comment:
We are making it a point to keep the unit the user enters, so if they choose to use bytes, then they will always see bytes. They can go back in and change it to megs or gigs or whatever they prefer.